packages/xdl/src/Project.ts

```diff
@@ -381,20 +381,35 @@ export async function getLatestReleaseAsync(
     owner?: string;
   }
 ): Promise<Release | null> {
-  // TODO(ville): move request from multipart/form-data to JSON once supported by the endpoint.
-  let formData = new FormData();
-  formData.append('queryType', 'history');
-  formData.append('slug', await getSlugAsync(projectRoot));
-  if (options.owner) {
-    formData.append('owner', options.owner);
-  }
-  formData.append('version', '2');
-  formData.append('count', '1');
-  formData.append('releaseChannel', options.releaseChannel);
-  formData.append('platform', options.platform);
-  const { queryResult } = await Api.callMethodAsync('publishInfo', [], 'post', null, {
-    formData,
-  });
+  let result;
+  if (process.env.EXPO_LEGACY_API === 'true') {
+    // TODO(ville): move request from multipart/form-data to JSON once supported by the endpoint.
+    let formData = new FormData();
+    formData.append('queryType', 'history');
+    formData.append('slug', await getSlugAsync(projectRoot));
+    if (options.owner) {
+      formData.append('owner', options.owner);
+    }
+    formData.append('version', '2');
+    formData.append('count', '1');
+    formData.append('releaseChannel', options.releaseChannel);
+    formData.append('platform', options.platform);
+    result = await Api.callMethodAsync('publishInfo', [], 'post', null, {
+      formData,
+    });
+  } else {
+    const user = await UserManager.ensureLoggedInAsync();
+    const api = ApiV2.clientForUser(user);
+    result = await api.postAsync('publish/history', {
+      owner: options.owner,
+      slug: await getSlugAsync(projectRoot),
+      version: '2',
+      releaseChannel: options.releaseChannel,
+      count: '1',
```

Comment:
Is there a reason why we use the string `'1'` instead of the number `1` here?

Reply:
I just copied line 392. Do you prefer 1?

Reply:
Line 392
https://github.com/expo/expo-cli/blob/d97f20b9d080434a5f0247b61bc64982b74f2cdf/packages/xdl/src/Project.ts#L392
used the string because the V1 publish endpoint took thzput as `multipart/form-data`, which, unlike JSON, only supports strings and not numbers.

I think `1` is better because it's more self-describing for numeric data.